docs/request-for-seed-provisioning.md

```diff
@@ -0,0 +1,35 @@
+# Request for SEED provisioning using TechPass portal
+
+The following table tells who can complete the steps mentioned in this article to request for SEED provisioning using the TechPass portal.
+
+ | User  | Device | Active TechPass | TechPass ID's domain format | Request for SEED provisioning |
+| ------------- |:-------------| :------------- | :------------- | ------ |
+| Public officer      | non-SE GSIB | Yes | *\<acronym-for-your-agency\>.gov.sg*. <br><br>For example, *tech.gov.sg*.| [TechPass portal](https://portal.techpass.gov.sg) |
+| Public officer | SE GSIB | Yes| *\<acronym-for-your-agency\>.gov.sg*. <br><br>For example, *tech.gov.sg*. | Create a [service request](https://go.gov.sg/techpass-sr) |
+| Vendor | non-SE GSIB | Yes | *\<acronym-for-your-engaging-agency\>.gov.sg*. <br><br>For example, *tech.gov.sg*.| [TechPass portal](https://portal.techpass.gov.sg) |
```

Comment:
this category isn't exactly right.. can we leave it out for now?

Reply:
so can i just say any user with an active TP and using a non-SE GSIB device can follow the steps to request for SEED provisioning?